app/views/assessment/endScreens/Share.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
+import env from 'react-native-config';
```

Comment:
I think we are using `Config` elsewhere in the app for this, I personally prefer the `env`, but wanted to point this out if you have thoughts on being consistent. As in should we change elsewhere from Config to env?

Reply:
We liked env better too. I like that change - will take care of that.